BREAKING.md

```diff
@@ -1,3 +1,336 @@
+  * [List Header](#list-header)
+  * [Menu](#menu)
+  * [Nav Link](#nav-link)
+  * [Searchbar](#searchbar)
+  * [Segment](#segment)
+  * [Skeleton Text](#skeleton-text)
+  * [Split Pane](#split-pane)
+  * [Toast](#toast)
+- [Colors](#colors)
+- [Events](#events)
+- [Mode](#mode)
+- [Ionicons](#ionicons)
+
+
+### CSS
+
+#### CSS Utilities
+
+We originally added CSS utility attributes for styling components because it was a quick and easy way to wrap text or add padding to an element. Once we added support for multiple frameworks as part of our Ionic for everyone approach, we quickly determined there were problems with using CSS attributes with frameworks that use JSX and Typescript. In order to solve this we added CSS classes. Rather than support CSS attributes in certain frameworks and classes in others, we decided to remove the CSS attributes and support what works in all of them, classes, for consistency. In the latest version of Ionic 4, there are deprecation warnings printed in the console to show what the new classes are, and the documentation has been updated since support for classes was added to remove all references to attributes: https://ionicframework.com/docs/layout/css-utilities.
+
+Some examples of what's changed are below. *This is not all-inclusive, see the documentation linked above for all of the available CSS utility classes.*
+
+```html
+<ion-header text-center></ion-header>
+<ion-content padding></ion-content>
+<ion-label text-wrap></ion-label>
+<ion-item wrap></ion-item>
+```
+
+becomes
+
+```html
+<ion-header class="ion-text-center"></ion-header>
+<ion-content class="ion-padding"></ion-content>
+<ion-label class="ion-text-wrap"></ion-label>
+<ion-item class="ion-wrap"></ion-item>
+```
+
+
+#### Display Classes
+
+The responsive display classes found in the `display.css` file have had their media queries updated to better reflect how they should work. Instead of using the maximum value of the breakpoint (for `.ion-hide-{breakpoint}-down` classes) the maximum of the media query will be the minimum of that breakpoint.
+
+
+#### Activated, Focused, Hover States
+
+The `.activated` class that gets added has been renamed to `.ion-activated` for consistency with how we add focused to elements and to avoid conflicts in user’s CSS.
+
+<!-- TODO mention some of the changes to the hover values: https://github.com/ionic-team/ionic/pull/19440 -->
+
+#### Distributed Sass
+
+The `scss` files have been removed from `dist/`, CSS variables should be used to theme instead.
+
+
+### Components
+
+#### Anchor
+
+The `ion-anchor` component has been renamed to `ion-router-link` as this is a better description of which component it should be used with. This component should still only be used in vanilla and Stencil JavaScript projects. Angular projects should use an `<a>` and `routerLink` with the Angular router. See the [documentation for router-link](https://ionicframework.com/docs/api/router-link) for more information.
```

Comment:
It technically was renamed since all of the functionality is the same, we just added `ion-router-link` and deprecated `ion-anchor` before removing it so it would give users time to update: https://github.com/ionic-team/ionic/commit/e7cd197af79cdf87f04bc769e0367c7e93c0aa0b#diff-36f66a20486cf32eda71ad087e5998bcL43